tests/Sdk/Acceptance/Extension/Tasks/PhpStanTaskCest.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Sdk\Acceptance\Extension\Tasks;
+
+use PHPUnit\Framework\Assert;
+use SprykerSdk\Sdk\Tests\AcceptanceTester;
+
+class PhpStanTaskCest
+{
+    /**
+     * @var string
+     */
+    public const COMMAND = 'validation:php:static';
+
+    /**
+     * @param \SprykerSdk\Sdk\Tests\AcceptanceTester $I
+     *
+     * @return void
+     */
+    public function testSuccess(AcceptanceTester $I)
+    {
+        // Arrange
+        $I->cleanReports();
+
+        // Act
+        $process = $I->runSdkCommand([
+            static::COMMAND,
+            '--path=src/PhpStan/success',
+        ]);
+
+        // Assert
+        Assert::assertTrue($process->isSuccessful());
+    }
+
+    /**
+     * @param \SprykerSdk\Sdk\Tests\AcceptanceTester $I
+     *
+     * @return void
+     */
+    public function testFail(AcceptanceTester $I)
+    {
+        // Arrange
+        $I->cleanReports();
+
+        // Act
+        $process = $I->runSdkCommand([
+            static::COMMAND,
+            '--path=src/PhpStan/fail',
+        ]);
+
+        // Assert
+        Assert::assertFalse($process->isSuccessful());
+        Assert::assertNotEmpty($process->getOutput());
+        Assert::assertStringContainsString('Violations found in files', $process->getOutput());
+        Assert::assertStringMatchesFormat('%aProperty %s does not accept string%a', $process->getOutput());
```

Comment:
Please put %s for code snippets always in ` inside of error messages

    Property `%s` does not ...
    
would be the correct message to be used.
This allows easier understanding of e.g. also empty, as well as direct copy and paste into all markdown supported platforms.

Reply:
I found that this message is coming directly from the PhpStan and so this assertion is testing PhpStan itself, which we would not want to do. I think the assertion should be deleted. Thank you for pointing this out.